R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -1518,6 +1518,16 @@ test_that("column functions", {
   result <- collect(select(df, arrays_overlap(df[[1]], df[[2]])))[[1]]
   expect_equal(result, c(TRUE, FALSE, NA))
 
+  # Test array_join()
+  df <- createDataFrame(list(list(list("Hello", "World!"))))
+  result <- collect(select(df, array_join(df[[1]], "#")))[[1]]
+  expect_equal(result, "Hello#World!")
+  df2 <- createDataFrame(list(list(list("Hello", NA, "World!"))))
```

Comment:
How does it work with NULL?

Reply:
@HyukjinKwon Thank you very much for your review. I will add a test for NULL and also change the }.